apps/web/src/lang/fr.json

```diff
@@ -6067,7 +6067,7 @@
     "description": "Title for the list of team members page"
   },
   "Vz719J": {
-    "defaultMessage": "Learn more about how GC Digital Talent handles user privacy.",
+    "defaultMessage": "Apprenez-en advantage sur la façon don’t le site GC Talents numériques traite de la vie privée des utilisateurs.",
```

Comment:
GC Digital Talent has been translated in French the past not as GC Talents numériques but as Talents numériques du GC. Has something changed?

Reply:
I don't think so. These are just the translations I got back. If this is the way we've been translated then I can and change it to that.

Reply:
Fixed by [bf00490](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9143/commits/bf0049005f1ad5021e6ca7a1b11b3b7aaf5548a8)